spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnSelector.scala

```diff
@@ -28,7 +28,10 @@ case class SomeColumns(columns: ColumnRef*) extends ColumnSelector {
   }.toMap
 
   override def selectFrom(table: TableDef): IndexedSeq[ColumnRef] = {
-    val missing = table.missingColumns(columns).filterNot(_ == RowCountRef)
+    val missing = table.missingColumns(columns).filterNot {
+      case _: FunctionCallRef => true
```

Comment:
As discussed at IRC channel, driver functionality providing functions inventories isn't available for Cassandra 2.1